src/main/java/banking/backend/Money.java

```diff
@@ -110,7 +143,9 @@ public boolean equals(Object o) {
      * @return a new instance after calculation
      */
     Money applyPercentage(Percentage percentage) {
-        throw new NotYetImplementedException();
+        Money money = new Money(0);
+        money.cents = this.cents * (percentage.getPercentage() / 100) / 100;
+        return money;
     }
```

Comment:
Method doesn't work as expected, Please change it to perform the operation = money * (100% + Percentage)